lib/units/auth/openid.js

```diff
@@ -0,0 +1,89 @@
+var http = require('http')
+var urljoin = require('url-join')

```

Comment:
Add new line above this. 

Basically, we follow this rule. 
- Build-In module goes in the top group
- 3rd party module in second group
- local module in third group